Translation corrections (#140)
07th-mod/higurashi-console-arcs: Update/some_012.txt

//rそこにはおかーさんが、足元に藤堂くんを見下ろしていて。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そこにはおかーさんが、足元に藤堂くんを見下ろしていて。",

> 			NULL, "Toudou-kun was lying on the ground, Mom looking down on him.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Toudou-kun was lying on the ground, with Mom looking down at him.", Line_Normal);
```

//rそれを見た瞬間、…私の中で何かの枷が外れて、心と頭の中が、急にすう…と静まり返っていった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それを見た瞬間、…私の中で何かの枷が外れて、心と頭の中が、急にすう…と静まり返っていった。",

> 			NULL, "The moment I saw that... the restraints in me were unleashed, and my heart and mind suddenly went quiet.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The moment I saw that... the restraints within me came unleashed, and my heart and mind suddenly went quiet.", Line_Normal);
```
